src/components/StyleGuide/ColorCard.tsx

```diff
@@ -7,38 +7,34 @@ import SimpleGrid from "../Grid/SimpleGrid";
 import Table from "../Table/Table";
 import Text from "../Text/Text";
 
-export interface ColorCardProps {
-  /** The backgroundColor of the color card. */
-  backgroundColor: string;
-  /** The name of a color's javascript theme object. */
-  colorName: string;
-  /** The name of the color that the current color is based on. */
-  colorSource: string;
+export interface DataTableProps {
   /** Contrast and WCAG compliance data related to the color black when used
    * with the current color. */
   dataBlack: string[];
   /** Contrast and WCAG compliance data related to the color white when used
    * with the current color. */
   dataWhite: string[];
+  dataHeading?: string[];
```

Comment:
I could use some explanation for these rows because just looking at them in the Figma file wasn't clear enough.

Reply:
```
/** Contrast and WCAG compliance data related to the default color for 
 * heading elements when used with the current color. */
dataHeadingColor?: string[];

/** Contrast and WCAG compliance data related to the default color for basic 
 * text elements when used with the current color. */
dataTextColor?: string[];
```

Please note that `body` was changed to `text` in the prop name.  Please do the same for the text that is rendered. in the card itself.

Reply:
So the `body` row title in the table will now be `text`? Is this what you mean? Will this be updated in Figma as well?

Reply:
Yes and yes.

Reply:
Maybe `dataWhite` and `dataBlack` should end with `Color` to be consistent. What do you think?

Reply:
> Maybe `dataWhite` and `dataBlack` should end with `Color` to be consistent. What do you think?

Yes, that’s a good idea.